integration_test.go

```diff
@@ -39,70 +39,104 @@ func TestIntegrationCartographer(t *testing.T) {
 	logger := logging.NewTestLogger(t)
 
 	cases := []struct {
-		description string
-		online      bool
-		imuEnabled  bool
-		mode        cartofacade.SlamMode
-		subAlgo     viamcartographer.SubAlgo
+		description     string
```

Comment:
@randhid We currently only do offline testing for lidar only as the online case for imu + lidar is more of a stress test than the offline case would be. If you would like every possible combination we can support this by changing the case structure to for loops that cover all possible combination of online [true, false], imuEnabled [tru, false]. odomEnabled [true, false].

Reply:
We'd have to refactor the integration tests for this, though, because offline only works for the lidar, not for the lidar + imu or odom or both.

Reply:
Refactoring would be simple, it would just involve skipping the blocking loop in the integration lidar/imu

Reply:
Ah, you're right!